docs/user-guide/map-templates.md

```diff
@@ -0,0 +1,66 @@
+# Map Templates
+*******************
+
+The **Map Templates** is an extension added to the context to browse the map template like the *WMC templates* and the *MapStore JSON*. The <img src="../img/button/map-templates-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provide to the user the list of the available templates. 
+
+<img src="../img/map-templates/map-templates-panel.jpg" class="ms-docimage"/>
+
+For each template in the *Map Templates* list the following buttons are displayed:
+
+<img src="../img/map-templates/map-templates-buttons.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+* The **Replace** button <img src="../img/button/replace-button.jpg" class="ms-docbutton" /> allows the user to replace the current map context with the template, as follows:
+
+<img src="../img/map-templates/replace-map.gif" class="ms-docimage"/>
+
+* The **Add Template** button <img src="../img/button/add_to_map_button.jpg" class="ms-docbutton" /> allows the user to add the template to current map context, as follows:
```

Comment:
The **Add Template** button <img src="../img/button/add_to_map_button.jpg" class="ms-docbutton" /> allows the user to add the map template contents (layers) to current map without replacing it (by default a new group is created in that case in TOC, on top of the other ones, to contains layers coming from the template to better identify them), as follows: